src/presentational-components/myUserAccess/bundles.js

```diff
@@ -0,0 +1,65 @@
+export const bundleData = [
```

Comment:
Yeah that's a really good idea. What are your thoughts about spinning up a config file/repo somewhere that holds that information? We could consume it here as well as in the other ones (for the JSON portions)

Reply:
I'd rather use some alrady working repository for this, like CSC (new file in this repository, that will be accessible under `/config/${fileName}`) or some other place. It could be chrome (but that's no ideal as it can't update based on user needs).